scripts/docker/tasking-manager/Dockerfile

```diff
@@ -4,18 +4,13 @@ FROM python:${PYTHON_IMG_TAG}-bookworm
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
-ENV PATH="/usr/src/app/__pypackages__/${PYTHON_IMG_TAG}/bin:$PATH" \
```

Comment:
why is this being removed?

Reply:
@mahesh-naxa

Reply:
@eternaltyro python stuff are inherited from python:3.10-bookworm base image. idk why `PATH` and `PYTHONPATH` are updated to use `/usr/src/app/__pypackages__/${PYTHON_IMG_TAG}/bin` where nothing exists. 
It was also flagged by sonarcube. these changes are non destructive.